config/bootstrap.php

```diff
@@ -99,6 +103,15 @@
 date_default_timezone_set('UTC');
 
 /**
+ * Sets the default output timezones.
+ * This output timezone is also being used by TimeHelper.
+**/
+Time::setDefaultOutputTimezone(Configure::read('App.defaultOutputTimezone'));
+Date::setDefaultOutputTimezone(Configure::read('App.defaultOutputTimezone'));
+FrozenTime::setDefaultOutputTimezone(Configure::read('App.defaultOutputTimezone'));
+FrozenDate::setDefaultOutputTimezone(Configure::read('App.defaultOutputTimezone'));
```

Comment:
Is this the right time to do this? I would expect that this would be based on user preferences, and we aren't far enough into the application to access user specific preferences just yet.